Sources/OktaOidc/iOS/OktaOidc+BrowserIOS.swift

```diff
@@ -19,9 +19,23 @@ extension OktaOidc: OktaOidcBrowserProtocolIOS {
 
     @objc public func signInWithBrowser(from presenter: UIViewController,
                                         callback: @escaping ((OktaOidcStateManager?, Error?) -> Void)) {
+        signInWithBrowser(from: presenter, payload: [:], callback: callback)
+    }
+    
+    @objc public func signInWithBrowser(from presenter: UIViewController,
+                                        payload: [String:String],
```

Comment:
Not clear what that parameter means. Maybe we can find better naming, something like: `oidcConfigExtension`, `oidcTransactionConfig`, `additionalOIDCParameters` and etc.

Reply:
I chose this name for consistency with the similar functionality present in the Android version.  Ideally I'd call it `additionalParameters` or something of the sort.  What do you think @IldarAbdullin-okta, is API consistency important enough? Otherwise I'll change this parameter name.

Reply:
Two SDKs are very different in implementations
I would implement what is better(and more clear) for iOS platform. I think `additionalParameters` is better naming